src/less/phone-input/phone-input.less

```diff
@@ -0,0 +1,183 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.phone-input {
+    display: inline-flex;
+}
+
+.phone-input button.btn {
+    min-width: 64px;
+    outline: none;
+    padding-inline: var(--spacing-200) var(--spacing-100);
+}
+
+.phone-input button.btn svg.icon:last-child {
+    margin-inline-start: var(--spacing-50);
+}
+
+.phone-input .listbox-button svg.flag {
+    height: 18px;
+    width: 24px;
+}
+
+.phone-input .listbox-button__value,
+.phone-input .listbox__value {
+    display: inline-flex;
+
+    & svg {
+        margin-inline-end: var(--spacing-200);
+    }
+
+    & span {
+        margin-inline-end: var(--spacing-100);
+    }
+
+    & span:last-child {
+        color: var(--color-foreground-secondary);
+    }
+}
+
+.phone-input .listbox-button__value span:nth-of-type(1),
+.phone-input .listbox__value span:nth-of-type(1) {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.phone-input .textbox {
+    align-items: center;
+    .background-color-token(textbox-background-color, color-background-secondary);
+    .border-color-token(textbox-border-color, color-stroke-default);
+    border-radius: 0 var(--border-radius-50) var(--border-radius-50) 0;
+    border-style: solid;
+    border-width: 1px;
+    color: var(--textbox-prefix-color, var(--color-foreground-primary));
+    display: inline-flex;
+    margin-left: -1px;
+    padding-inline-start: var(--spacing-100);
+}
+
+.phone-input .textbox > input.textbox__control {
+    border: none;
+    height: 36px;
+    padding-inline: var(--spacing-50);
+}
+
+.phone-input .textbox > input.textbox__control:focus {
+    .background-color-token(textbox-focus-background-color, color-background-secondary);
+    outline: none;
+}
+
+// stylelint-disable no-descending-specificity
+.phone-input button.btn:focus,
+.phone-input:not(.phone-input--readonly) .textbox:focus-within {
+    .border-color-token(textbox-focus-border-color, color-stroke-strong);
+    box-shadow: 0 0 0 0.5pt var(--color-stroke-strong);
+    position: relative;
+    z-index: 1;
+}
+
+.phone-input--large {
+    height: 48px;
+}
+
+.phone-input--fluid,
+.phone-input--fluid .textbox,
+.phone-input--fluid .textbox > input.textbox__control {
+    width: 100%;
+}
+
+.phone-input--readonly .listbox-button button[aria-disabled="true"],
+.phone-input--readonly .listbox-button button[disabled] {
+    .border-color-token(textbox-border-color, color-stroke-default);
+}
```

Comment:
Two issues with this
1. There is no focus indicator when tabbing through
2. The listbox button is disabled and therefore not in the tab index. This is fine if its disabled but for readonly, theres no way for screanreaders to see what the selected country is. I dunno if this is important, but it might be.

Reply:
This is kind of expected because we are using disabled to make it read only for a button (technically button cannot be read only). So, my assumption is SR user can use other keys to navigate to previous element and figure out that button is dimmed with country code. @ianmcburnie any thoughts?

Reply:
Yes, @saiponnada is correct. Screen reader users can still access the button even if disabled.